pkg/yurthub/proxy/remote/remote.go

```diff
@@ -106,21 +110,13 @@ func (rp *RemoteProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	if httpstream.IsUpgradeRequest(req) {
 		klog.V(5).Infof("get upgrade request %s", req.URL)
 		if isBearerRequest(req) {
```

Comment:
As mentioned in https://github.com/openyurtio/openyurt/pull/683/files#r769771572,  if we group `upgradeAwareHandler` and `bearerUpgradeAwareHandler` into one, the codes here can be simplified as follow:
```
	if httpstream.IsUpgradeRequest(req) {
		klog.V(5).Infof("get upgrade request %s", req.URL)
		rp.upgradeHandler.ServeHTTP(rw, req)
		return
	}

	rp.reverseProxy.ServeHTTP(rw, req)
```

Reply:
As replied above